musicbrainz/csv/convert_to_csv.py

```diff
@@ -120,13 +130,13 @@ def extract(data, value: dict, first_level: bool = True, key: str = ""):
 
     else:
         # if data is not a collection, we parse it and add to the current value dictionary.
-        global header
         if key not in header:
             header.append(key)
 
         v = data
         if isinstance(data, str):
-            v = v.replace(",", "\,")
+            if "," in v:
```

Comment:
You’re working way too hard. The Python CSV writer auto-escapes values with commas.